File: sync/src/main/scala/com/fan/mysql/column/EventColumn.java
package com.fan.mysql.column;

import java.io.Serializable;

public class EventColumn {

	private Serializable columnValue;
	private boolean isNull = false;

	public EventColumn() {
	}

	public Serializable getColumnValue() {
		return columnValue;
	}

	public void setColumnValue(Serializable columnValue) {
		this.columnValue = columnValue;
	}

	public boolean isNull() {
		return isNull;
	}

	public void setNull(boolean isNull) {
		this.isNull = isNull;
	}

}

File: async/src/main/scala/com/fan/mysql/async/binlog/event/EventColumn.scala
package com.fan.mysql.async.binlog.event

import java.io.Serializable

import org.apache.commons.lang3.builder.{ToStringBuilder, ToStringStyle}

class EventColumn {
  private var columnValue: Serializable = _

  private var isNull = false

  def getColumnValue: Serializable = columnValue

  def setColumnValue(columnValue: Serializable): Unit = {
    this.columnValue = columnValue
  }

  def setNull(isNull: Boolean): Unit = {
    this.isNull = isNull
  }

  override def toString: String =
    ToStringBuilder.reflectionToString(this, ToStringStyle.DEFAULT_STYLE)

}
